### ISSUE TITLE
Question: measured time for ZOO models

### ISSUE THREAD

**Issue by Borda** [tensorflow/models#7306]
@pkulzc I was playing around the pre-trained model for object detection, and very appreciate your comparison in [Tensorflow detection model zoo](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md). I am mostly using the following notebook [
Object Detection Demo](https://github.com/tensorflow/models/blob/master/research/object_detection/object_detection_tutorial.ipynb) running in [Gollab](https://colab.research.google.com/drive/1jUJaCzgtakpIu2wPnSJcRoNcRobC8d_C). I am not able to get anywhere close to your reported times even using TPU, my execution time is around 7s per image which is about 100x slower... I see that you have used GTX TITAN X card, but still is TPU so much worse? Any idea what is happening?

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by Borda** [comment]
You should ask Google, this information is not provided...
```
posix.uname_result(sysname='Linux', nodename='8f1336f5b67f', release='4.14.79+', version='#1 SMP Wed Dec 19 21:19:13 PST 2018', machine='x86_64')
print(tf.GIT_VERSION, tf.VERSION) >> v1.14.0-rc1-22-gaf24dc91b5 1.14.0
```
as reccomended I have tried to run
```
! wget -q https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh
! chmod +x tf_env_collect.sh
! ./tf_env_collect.sh
```
but it fails with:
```
./tf_env_collect.sh: line 7: syntax error near unexpected token `newline'
./tf_env_collect.sh: line 7: `<!DOCTYPE html>'
```

**Comment by Borda** [comment]
@tensorflowbutler @gowthamkpr any update here?

**Comment by tombstone** [comment]
@Borda  It might be due to how you are measuring it. There is a startup cost associated with setting up model and running inference. We set up the graph first, do a few warmup runs and then take the mean of 100 runs or so. The cost also depends on the size of the images you are pushing through the model. If this doesn't answer your question, please provide details of how you are measuring the latency and we can try to address it.
